deploy/examples/operator.yaml

```diff
@@ -61,6 +61,10 @@ data:
   # Supported values from 0 to 5. 0 for general useful logs (the default), 5 for trace level verbosity.
   # CSI_SIDECAR_LOG_LEVEL: "0"
 
+  # csi driver name prefix for cephfs, rbd and nfs. if not specified, default
+  # will be the namespace name where rook-ceph operator is deployed.
+  # CSI_DRIVER_NAME_PREFIX: "rook-ceph"
```

Comment:
user also need to use this name while creating the storageclass provisioner, so we should provide instructions for that.

Reply:
Yes correct, will update the corresponding doc

Reply:
@travisn adding comment here is enough or should we update in all the other places where we have storageclass yaml and Readme docs?

Reply:
In the storage classes, how about if we add something that can be easily replaced? For example:
```
provisioner: rook-ceph.rbd.csi.ceph.com # csidriver:rbd-name
```

This is similar to other places where we have `namespace:cluster` or `namespace:operator`. 

Then we can document in two places to search and replace these names:
1. Here in the operator.yaml with this new setting
2. In ceph-configuration.md with the "Using alternate namespae" topic. Today the prefix is not actually changed with that documentation.

Reply:
@travisn i updated the helm chart to take new value to consideration and also update the CSI driver readme with new section for setting the custom name but not updated the yaml files. i assume as this is advance feature i assume people will try it out as per the doc. Let me know if this still need to be updated.

Reply:
Perfect for automating in the helm chart settings. For the non-helm settings, the driver name is still a challenge for users to update because we don't have something they can easily search for to replace. Even if we just add this generic comment to all the examples that have a provisioner name, at the end of the provisioner line. Then we can mention simply to search-and-replace those lines, both in that topic you created and in the alternate namespace topic.
`# csi-provisioner-name`

Reply:
@travisn i think it took more work then expected as it needs to be done, PTAL and let me know if more changes are required

Reply:
I was not trying to suggest we need a script command to automate it for them, just to give them a string to search for themselves. :)